packages/nimbus/src/patterns/fields/money-input-field/money-input-field.dev.mdx

```diff
@@ -0,0 +1,763 @@
+  });
+
+  return (
+    <Stack direction="column" gap="400">
+      <MoneyInputField
+        label="US Dollars (2 decimals)"
+        description="Standard 2 decimal places"
+        currencies={["USD"]}
+        value={usd}
+        onValueChange={setUsd}
+      />
+      <MoneyInputField
+        label="Japanese Yen (0 decimals)"
+        description="Whole numbers only"
+        currencies={["JPY"]}
+        value={jpy}
+        onValueChange={setJpy}
+      />
+      <MoneyInputField
+        label="Kuwaiti Dinar (3 decimals)"
+        description="High precision with 3 decimal places"
+        currencies={["KWD"]}
+        value={kwd}
+        onValueChange={setKwd}
+      />
+    </Stack>
+  );
+}
+```
+
+## Component requirements
+
+### Value type
+
+MoneyInputField uses the `MoneyInputValue` type:
+
+```tsx
+type MoneyInputValue = {
+  amount: string;           // String representation with decimal (e.g., "123.45")
+  currencyCode: CurrencyCode | "";  // ISO currency code or empty string
+}
+
+type CurrencyCode = "USD" | "EUR" | "GBP" | "JPY" | ... // 100+ currencies
+```
+
+**Important**:
+- The `amount` is always a string, not a number
+- Use a dot (`.`) as the decimal separator
+- Empty amount should be `""` (empty string)
+- The component handles currency-specific formatting automatically
+
+### Dual input nature
+
+MoneyInputField contains **two focusable inputs**:
+1. **Currency selector** (combobox) - For choosing the currency
+2. **Amount input** (spinbutton) - For entering the monetary value
+
+Both inputs receive separate IDs based on the field's `id` or `name` prop:
+- Amount: `${id}.amount` or `${name}.amount`
+- Currency: `${id}.currencyCode` or `${name}.currencyCode`
+
+### Error handling
+
+MoneyInputField uses the `FieldErrors` component to display validation errors with built-in localized messages. For custom error messages, use the `renderError` prop (see [Custom error rendering](#custom-error-rendering) below).
+
+**Error object format** (`FieldErrorsData`):
+```tsx
+type FieldErrorsData = {
+  [key: string]: boolean | string | ReactNode;
+};
+
+// Examples:
+const errors = {
+  missing: true,              // Built-in message: "This field is required"
+  invalidFormat: true,        // Built-in message: "Please enter a valid format"
+  custom: "Custom error text" // Custom message
+};
+```
+
+### Accessibility
+
+The MoneyInputField handles most accessibility requirements internally through FormField and MoneyInput. The `label` prop is required and ensures proper labeling for screen readers.
+
+If your use case requires tracking and analytics for this component, it is good practice to add a **persistent**, **unique** id:
+
+```tsx
+const PERSISTENT_ID = "product-price-input";
+
+export const Example = () => (
+  <MoneyInputField
+    id={PERSISTENT_ID}
+    label="Product Price"
+    currencies={["USD", "EUR", "GBP"]}
+    value={value}
+    onValueChange={setValue}
+  />
+);
+```
+
+#### Keyboard navigation
```

Comment:
I like that this is its own section. The other two patterns are not like this and will need to be updated.